tests/validation/tests/v3_api/test_workload.py

```diff
@@ -861,24 +862,33 @@ def setup_project_by_role(role, remove_resource):
     else:
         return None, None, None, None
 
-# --------------------- rbac tests by workload type -----------------------
+
+# --------------------- rbac tests by workload types -----------------------
 
 @if_test_rbac
 @pytest.mark.parametrize("role", RBAC_ROLES)
-def test_rbac_wl_daemonset_create(role, remove_resource):
+@pytest.mark.parametrize("config", WORKLOAD_TYPES)
+def test_rbac_wl_parametarize_create(role, config, remove_resource):
     p_client, project, ns = setup_daemonset_project_by_role(role)
     cluster = namespace["cluster"]
     con = [{"name": "test1", "image": TEST_IMAGE}]
     name = random_test_name("default")
     if role != PROJECT_READ_ONLY:
-        workload = p_client.create_workload(name=name,
-                                            containers=con,
-                                            namespaceId=ns.id,
-                                            daemonSetConfig={})
+        workload = create_workload_by_type(p_client, name, con, ns, config)
         wait_for_wl_to_active(p_client, workload)
         schedulable_node_count = len(get_schedulable_nodes(cluster))
-        validate_workload(p_client, workload, "daemonSet",
-                          ns.name, schedulable_node_count)
+        if config == daemonSetConfig:
+            validate_workload(p_client, workload, "daemonSet",
+                              ns.name, schedulable_node_count)
+        elif config == statefulSetConfig:
+            validate_workload(p_client, workload, "statefulSet",
+                              ns.name, schedulable_node_count)
+        elif config == chronJobConfig:
+            validate_workload(p_client, workload, "cronJob",
+                              ns.name, schedulable_node_count)
+        elif config == jobConfig:
+            validate_workload(p_client, workload, "job",
+                              ns.name, schedulable_node_count)
```

Comment:
If we change 
```
WORKLOAD_TYPES = ["daemonSetConfig", "statefulSetConfig", "cronJobConfig", "jobConfig"]
````
to 
```
WORKLOAD_TYPES = ["daemonSet", "statefulSet", "cronJob", "job"]
```

we don't have to invoke validate_workload four times here.

Reply:
Addressed in changes.